staking/availability/measure.go

```diff
@@ -126,6 +128,17 @@ func bumpCount(
 			continue
 		}
 
+		snapshot, err := bc.ReadValidatorSnapshot(addr)
+		if err != nil {
+			return err
+		}
+
+		if snapshot.LastEpochInCommittee.Cmp(common.Big0) == 0 {
```

Comment:
The snapshot is what the validator was from last epoch. If it's new validator in last epoch, it doesn't affect that we should still count his signing in this epoch if it is already selected in the committee.

Reply:
Because I noticed that it started counting one epoch too early because you have all validators written in snapshot , and maybe did have this epoch as 0, so I prefer keeping this guard

Reply:
actually you're right